_posts/2021-08-31-monero-GUI-0.17.2.3-released.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Monero GUI 0.17.2.3 'Oxygen Orion' released"
+summary: Point release containing decoy selection bug fix that improves privacy.
+tags: [releases]
+author: selsta
+image: /blog/assets/2020-09-17-monero-0.17-released/oxygen-orion.png
+---
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+# Overview
+
+This is the v0.17.2.3 minor point release of the Monero GUI software. This release contains a decoy selection bug fix that improves privacy.
+
+[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/04/09/monero-0.17.2.0-released.html)
```

Comment:
```suggestion
[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/04/09/monero-0.17.2.0-released.html).
```

Reply:
Fixed the link from v0.17.2.0 to v0.17.2.3 and added missing dot.